mapie/classification.py

```diff
@@ -1109,6 +581,7 @@ def fit(
         """
         # Checks
         (estimator,
+         self.conformity_score_function_,
```

Comment:
Just a few lines bellow you check `if self.method == "raps"` but if we use the conformity score RAPS() then self.method will be None. In this case this piece of code won't work if `conformity_score=RAPS()` (maybe a test is missing)

Reply:
Tell me if the proposed solution solves this problem:
- everything that depends on the logic of the RAPS code is included in the eponymous class.
- everything that depends on checking that the correct values have been assigned is managed in the `utils.py` file in the `conformity_scores` module.

Reply:
Done 👍